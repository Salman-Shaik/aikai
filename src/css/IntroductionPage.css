.introduction {
  padding: 4vh 1vw;
  text-align: center;
  width: 65%;

  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: space-around;
  background-image: repeating-linear-gradient(
      157.5deg,
      rgb(34, 34, 34) 0px,
      rgb(34, 34, 34) 10px,
      transparent 10px,
      transparent 11px
    ),
    repeating-linear-gradient(
      67.5deg,
      rgb(34, 34, 34) 0px,
      rgb(34, 34, 34) 10px,
      transparent 10px,
      transparent 11px
    ),
    linear-gradient(90deg, rgb(52, 52, 52), rgb(52, 52, 52));
  background-image: -webkit-repeating-linear-gradient(
      157.5deg,
      rgb(34, 34, 34) 0px,
      rgb(34, 34, 34) 10px,
      transparent 10px,
      transparent 11px
    ),
    repeating-linear-gradient(
      67.5deg,
      rgb(34, 34, 34) 0px,
      rgb(34, 34, 34) 10px,
      transparent 10px,
      transparent 11px
    ),
    linear-gradient(90deg, rgb(52, 52, 52), rgb(52, 52, 52));
}

.introduction_page {
  display: flex;
  align-items: center;
  justify-content: center;
  min-height: 89vh;
}

.heading {
  background-image: linear-gradient(
    to right bottom,
    #f14f2e,
    #f03f42,
    #ec3154,
    #e42764,
    #d82474
  );
  background-image: -webkit-linear-gradient(
    to right bottom,
    #f14f2e,
    #f03f42,
    #ec3154,
    #e42764,
    #d82474
  );
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
  text-align: center;
  font-size: 2.2em;
  font-weight: bold;
  margin-bottom: 1vh;
  cursor: default;
}

.info {
  background-image: -webkit-linear-gradient(-225deg, #e3fdf5 0%, #ffe6fa 100%);
  background-image: linear-gradient(-225deg, #e3fdf5 0%, #ffe6fa 100%);
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
  font-size: 1.05em !important;
  font-weight: normal;
  width: 62%;
  word-break: break-word;
  margin-bottom: 6vh;
  cursor: default;
}
