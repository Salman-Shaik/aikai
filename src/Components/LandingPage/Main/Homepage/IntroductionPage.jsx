import React from "react";
import "../../../../css/IntroductionPage.css";
import { LinkMenu } from "./LinkMenu";

export const IntroductionPage = () => {
  return (
    <section className="introduction_page">
      <section className="introduction">
        <span className="heading">Random Movie / TV Show Generator</span>
        <p className="info">
          Confused what to watch today, want to know the details of a movie/tv
          show, want to get recommendations for a movie/tv show, want to know
          what's playing in theatres or what's airing on TV. You can get all of
          them here @ A.I.K.A.I .
        </p>
        <LinkMenu />
      </section>
    </section>
  );
};
