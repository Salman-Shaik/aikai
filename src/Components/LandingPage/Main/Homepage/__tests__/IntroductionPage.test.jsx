import { render } from "@testing-library/react";
import React from "react";
import { IntroductionPage } from "../IntroductionPage";

describe("Introduction Page", () => {
  it("Snapshot Test", () => {
    const { container } = render(<IntroductionPage />);
    expect(container).toMatchSnapshot();
  });
});
