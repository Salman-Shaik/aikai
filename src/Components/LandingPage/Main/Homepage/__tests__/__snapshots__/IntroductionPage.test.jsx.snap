// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Introduction Page Snapshot Test 1`] = `
<div>
  <section
    class="introduction_page"
  >
    <section
      class="introduction"
    >
      <span
        class="heading"
      >
        Random Movie / TV Show Generator
      </span>
      <p
        class="info"
      >
        Confused what to watch today, want to know the details of a movie/tv show, want to get recommendations for a movie/tv show, want to know what's playing in theatres or what's airing on TV. You can get all of them here @ A.I.K.A.I .
      </p>
    </section>
  </section>
</div>
`;
